### PR TITLE
Add readme and codecov report

### PR DESCRIPTION
**Pull request by LakshanKarunathilake**
## Purpose
Update readme and codecov  support 

## Related Issues
https://github.com/wso2-enterprise/choreo/issues/12760

## Checklist
- [X] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,91 @@
...
+
+Ballerina Observe Internal Library
+===================
+
+[![Build](https://github.com/ballerina-platform/module-ballerina-http/actions/workflows/build-timestamped-master.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinai-observe/actions/workflows/build-timestamped-master.yml)
+[![Trivy](https://github.com/ballerina-platform/module-ballerina-http/actions/workflows/trivy-scan.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinai-observe/actions/workflows/trivy-scan.yml)
+[![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerina-http.svg)](https://github.com/ballerina-platform/module-ballerinai-observe/commits/master)
+[![codecov](https://codecov.io/gh/ballerina-platform/module-ballerina-observe/branch/master/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerina-observe)
+
+This module contains internal configurations and initializations for Ballerina Observability. Ballerina supports Observability out of the box. You can user [module-ballerina-observe](https://github.com/ballerina-platform/module-ballerina-observe) in your ballerina project and enable observability features.
```
Comment (praneesha) [inline README.md:10]:
```suggestion
This module contains internal configurations and initializations for Ballerina observability. Ballerina supports observability out of the box. You can use the [module-ballerina-observe](https://github.com/ballerina-platform/module-ballerina-observe) in your Ballerina project and enable observability features.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,91 @@
...
+
+1. Download and install Java SE Development Kit (JDK) version 11 (from one of the following locations).
+
+    * [Oracle](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html)
+
+    * [OpenJDK](https://adoptopenjdk.net/)
+
+      > **Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.
+
+2. Export GitHub Personal access token with read package permissions as follows,
```
Comment (praneesha) [inline README.md:26]:
```suggestion
2. Export your GitHub Personal access token with read package permissions as follows.
```

File: README.md (hunk 1 above)
Comment (LakshanKarunathilake) [reply README.md]: Fixed with https://github.com/ballerina-platform/module-ballerina-observe/pull/36/commits/64ab79975b04a60986006be9c42aec231db262d0

File: README.md (hunk 2 above)
Comment (LakshanKarunathilake) [reply README.md:27]: Fixed with https://github.com/ballerina-platform/module-ballerina-observe/pull/36/commits/64ab79975b04a60986006be9c42aec231db262d0

**Review by praneesha — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,2 +1,92 @@
...
+
+Ballerina Observe Internal Library
+===================
+
+[![Build](https://github.com/ballerina-platform/module-ballerina-http/actions/workflows/build-timestamped-master.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinai-observe/actions/workflows/build-timestamped-master.yml)
+[![Trivy](https://github.com/ballerina-platform/module-ballerina-http/actions/workflows/trivy-scan.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinai-observe/actions/workflows/trivy-scan.yml)
+[![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerina-http.svg)](https://github.com/ballerina-platform/module-ballerinai-observe/commits/master)
+[![codecov](https://codecov.io/gh/ballerina-platform/module-ballerina-observe/branch/master/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerina-observe)
+
+lerina project and enable observability features.
```
Comment (nadundesilva) [inline README.md:10]: There seems to be something wrong with this sentence 🤔

Comment (LakshanKarunathilake) [reply README.md]: Fixed with https://github.com/ballerina-platform/module-ballerina-observe/pull/36/commits/f1ac7a646e1fbb9c4d5546f7755da6069af062ea

**Review by nadundesilva — APPROVED** [review]

**Pull request merged**